.eslintrc

```diff
@@ -1,6 +1,7 @@
 {
   "parserOptions": {
-    "sourceType": "module"
+    "sourceType": "module",
+    "ecmaVersion": 2018
```

Comment:
Fix for `s` regex flag not recognised by our current ESLint configuration. We just need to tell the parser to expect 2018 ECMA features.

Reply:
Maybe we can set ecmaversion only in test/.eslintrc, which will only affect the test file.

Reply:
I didn't realise we had such file. I took the liberty to delete `test/.eslintrc` and put its content into the `overrides` property of the main ESLint configuration file:

```none
  "overrides": [
    {
      "files": ["test/**/*.js"],
      "parserOptions": {
        "ecmaVersion": 2018
      },
      "rules": {
        "no-restricted-properties": ["error", {
          "object": "describe",
          "property": "only"
        }],
        "no-restricted-modules": ["error", ".."]
      }
    }
  ]
```

I think this is the best of both worlds: we have only one configuration file to maintain whilst still allowing specific paths to override the configuration. 

So now the `parserOptions.ecmaVersion` is set to `2018` only when the linter runs on test files.

Does that work for you and everybody else?